packages/web/src/components/form-fields/TextAreaField.tsx

```diff
@@ -8,9 +8,11 @@ type TextAreaFieldProps = TextAreaV2Props & {
 
 export const TextAreaField = (props: TextAreaFieldProps) => {
   const { name, ...other } = props
-  const [field, meta] = useField(name)
+  const [{ value, ...field }, meta] = useField(name)
 
   const hasError = Boolean(meta.touched && meta.error)
 
-  return <TextAreaV2 {...field} error={hasError} {...other} />
+  return (
+    <TextAreaV2 value={value ?? ''} {...field} error={hasError} {...other} />
```

Comment:
lets not do this, instead ensure value is always controlled and at least empty string in the initialValues. lmk if that makes sense.